### PR TITLE
Send data to a kafka topic via the ex_admin panel

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Related PRs
Waiting for https://github.com/santiment/sanbase2/pull/1599 so I can reuse the refactored kafka exporter

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase_web/admin/kafka/kafka_label_record.ex [hunk 1]
```diff
@@ -0,0 +1,67 @@
+defmodule Sanbase.Model.Kafka.KafkaLabelRecord do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "kafka_label_records" do
+    field(:topic, :string)
+    field(:sign, :integer, default: 1)
```
Comment (kalkov) [inline lib/sanbase_web/admin/kafka/kafka_label_record.ex]: I think you don't need a default value, will be always present.

File: lib/sanbase_web/admin/kafka/kafka_label_record.ex [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    controller do
+      before_filter(:send_to_kafka, only: [:create])
+    end
+  end
+
+  def send_to_kafka(conn, %{kafka_label_record: kafka_label_record} = params) do
+    {topic, data} =
+      Map.pop(kafka_label_record, :topic)
+      |> IO.inspect(label: "45", limit: :infinity)
```
Comment (kalkov) [inline lib/sanbase_web/admin/kafka/kafka_label_record.ex]: the inspect looks like leftover

File: priv/repo/migrations/20191118125110_create_kafka_push_records_table.exs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+defmodule Sanbase.Repo.Migrations.CreateKafkaPushRecordsTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:kafka_label_records) do
+      add(:topic, :string)
```
Comment (kalkov) [inline priv/repo/migrations/20191118125110_create_kafka_push_records_table.exs]: Most of the fields should have not null.

File: config/config.exs [hunk 4]
```diff
@@ -19,8 +19,12 @@ config :ecto, json_library: Jason
 config :sanbase, Sanbase, environment: "#{Mix.env()}"
 
 config :sanbase, Sanbase.KafkaExporter,
+  supervisor: SanExporterEx.Producer.Supervisor,
+  producer: SanExporterEx.Producer,
   kafka_url: {:system, "KAFKA_URL", "blockchain-kafka-kafka"},
-  kafka_port: {:system, "KAFKA_PORT", "9092"}
+  kafka_port: {:system, "KAFKA_PORT", "9092"},
+  prices_topic: {:system, "KAFKA_PRICES_TOPIC", "asset_prices"},
```
Comment (tspenov) [inline config/config.exs]: yeah, that is indeed better 👍

File: lib/sanbase_web/admin/kafka/kafka_label_record.ex [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+          data
+          |> List.wrap()
+          |> Enum.map(&{"", Jason.encode!(&1)})
+
+        @producer.send_data(topic, data)
+
+        {conn, params}
+
+      false ->
+        {Phoenix.Controller.put_flash(
```
Comment (tspenov) [inline lib/sanbase_web/admin/kafka/kafka_label_record.ex]: nice!

File: lib/sanbase_web/admin/kafka/kafka_label_record.ex (hunk 1 above)
Comment (IvanIvanoff) [reply lib/sanbase_web/admin/kafka/kafka_label_record.ex]: Removed it.
It was added so it's prefilled in the table as I expect 1 to be the most used value.

**Pull request merged**